.github/workflows/ci.yml

```diff
@@ -177,6 +177,23 @@ jobs:
         chmod a+x builder
         ./builder build -p ${{ env.PACKAGE_NAME }} --cmake-extra=-DCMAKE_OSX_DEPLOYMENT_TARGET=10.9
 
+  freebsd:
+    runs-on: ubuntu-22.04 # latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build ${{ env.PACKAGE_NAME }} + consumers
+        uses: cross-platform-actions/action@v0.23.0
+        with:
+          operating_system: freebsd
+          architecture: x86-64
+          version: '14.0'
+          shell: bash
+          run: |
+            sudo pkg install -y python3 py39-urllib3
+            python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz', 'builder')"
+            chmod a+x builder
+            ./builder build -p ${{ env.PACKAGE_NAME }}
+
   openbsd:
     runs-on: macos-12 # macos's virtual machine is faster than ubuntu's
```

Comment:
May as well run the openbsd job on ubuntu as well (and update the action version) since that's what they recommend now.
> In general the Ubuntu runners are the preferred choice. Both macOS and Ubuntu runners support hardware accelerated nested virtualization. But the Ubuntu runners have more resources and therefore better performance.

Reply:
yeah, it has been merged, https://github.com/awslabs/aws-c-cal/pull/179